.github/workflows/pypi.yml

```diff
@@ -56,12 +56,12 @@ jobs:
         include:
           - os: ubuntu-latest
             name: linux-x86_64
-          - os: windows-2019
+          - os: windows-latest
             name: windows-amd64
-          - os: macos-11
+          - os: macos-13
             name: macos-x86_64
             macos_arch: "x86_64"
-          - os: macos-11
+          - os: macos-14
```

Comment:
@EwoutH, can you comment on these changes?  will these be needed for 3.13?

Reply:
Not needed an sich, but it's good to compile on the latest host OS versions with the latest toolsets, especially for arm64.

Reply:
huh, i was previously encouraged to compile platform specific stuff on the oldest compatible toolset...  am I misapplying/misunderstanding old advice?

Reply:
Generally you use the latest tooling / toolset / toolchain, but with, if needed, a certain target version, which guarantees compatible with that version and up. For example, CMake uses [`MACOSX_DEPLOYMENT_TARGET`](https://cmake.org/cmake/help/latest/envvar/MACOSX_DEPLOYMENT_TARGET.html) for this.

For Python I don't believe it really matters much.

Reply:
Yeah I remember futzing with macOS deployment target.  Can't remember if it was this project or another, but there was some issue with hatchling not always respecting that env var.  May have been fixed by now.

Anyway, let's just go with it and I'll revisit later if anything fails when it's time to release.  The main python functionality is all working fine on 3.13

Thanks again